EXT_NRAYTRACING.md

```diff
@@ -0,0 +1,114 @@
+Domain-Specific Naming Conventions for Neutron raytracing codes
+===============================================================
+
+openPMD extension name: `NRAYTRACING`
+
+
+Introduction
+------------
+
+This extention is designed for transfer of neutron particle states between neutron raytracing simulation codes.
+The neutrons are saved in rays with a given weight corresponding to expected intensity.
+This standard is not yet finished and is currently based on EXT_BeamPhysics / EXT_WAVEFRONT
+
+Each record needs to have the same length, as the n'th index for each record correspond to particle n.
+
+Coordinates are defined with z in the beam direction and if possible y in the direction opposite gravity.
+
+Particle Records
+----------------
+
+- `position/`

```

Comment:
ok for me, but if you see good reasons to store all vector components in one set, please revert to the previous.

Reply:
I think I have found a solution to my previous issue with optional vectors such as spin.

Current implementation:
- `spin/`
    - type: Optional 
    - description: Particle spin direction, no unit
    - `x` 
        - type: Optional *(real)*
        - description: x component of spin

Could be fixed by making the components required, but the entire spin group could remain optional. Would that work as expected?
- `spin/`
    - type: Optional 
    - description: Particle spin direction, no unit
    - `x` 
        - type: Required *(real)*
        - description: x component of spin